### PR TITLE
[BUGFIX] fix time series vertical positioning

### PR DESCRIPTION
**Pull request by jgbernalp**
This PR fixes the vertical positioning of the time series tooltip on a dashboard with several panels.

Issue:

https://github.com/perses/perses/assets/5461414/ef274d99-6696-4331-8ba1-9ef082a5b025

Fix:

https://github.com/perses/perses/assets/5461414/ad9da70f-0641-487a-86b6-e79f6d8f185a

### CODE REVIEW

File: ui/components/src/TimeSeriesTooltip/utils.ts [hunk 1]
```diff
@@ -40,17 +45,12 @@ export function assembleTransform(
...
-  if (isCloseToBottom === true) {
-    if (seriesNum > 6) {
-      y = mousePos.page.y * 0.75;
-    } else {
-      y = mousePos.page.y * 0.9;
-    }
-  } else if (mousePos.plotCanvas.y > yPosAdjustThreshold) {
-    y = mousePos.page.y * 0.95;
+  // approximate tootlip height based on number of series
+  const tooltipHeight = Math.min(Math.min(seriesNum, TOOLTIP_MAX_ITEMS) * 24, TOOLTIP_MAX_HEIGHT);
```
Comment (sjcobb) [inline ui/components/src/TimeSeriesTooltip/utils.ts:49]: Awesome, much better than the static `if (seriesNum > 6)` hack I did before, plus this actually fixes the core problem!

Comment (sjcobb) [reply ui/components/src/TimeSeriesTooltip/utils.ts:49]: nit: typo in comment that says `tootlip` instead of `tooltip`

**Review by sjcobb — APPROVED** [review]
🙏 thanks for fixing this!

**Comment by juliepagano** [comment]
I'm seeing some cases where the tooltip is cut off more often with this approach than the previous one when the mouse cursor is near the bottom of the viewport. @sjcobb -- you're a lot more familiar with this bit of code than me. Do you think there's some additional tuning that should be done before merging (maybe some adjustments to the height calc?) or is this still a significant improvement over what we had before?

## With this PR 
image

## With code from `main`
image

**Pull request merged**